extensions/2.0/Vendor/EXT_feature_metadata/1.0.0/README.md

```diff
@@ -0,0 +1,1665 @@
+                "value": 0
+              },
+              {
+                "name": "Commercial",
+                "value": 1
+              },
+              {
+                "name": "Hospital",
+                "value": 2
+              },
+              {
+                "name": "Other",
+                "value": 3
+              }
+            ]
+          }
+        }
+      },
+      "statistics": {
+        "classes": {
+          "building": {
+            "count": 100,
+            "properties": {
+              "height": {
+                "min": 3.9,
+                "max": 341.7,
+                "mode": 5.0
+              },
+              "buildingType": {
+                "occurrences": {
+                  "Residential": 70,
+                  "Commercial": 28,
+                  "Hospital": 2
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+## Examples
+
+_This section is non-normative_
+
+The examples below shows the breadth of possible use cases for this extension. 
+
+Example|Description|Image
+--|--|--
+Triangle mesh|Feature IDs are assigned to each vertex to distinguish components of a building.|![Building Components](figures/building-components.png)
+Per-vertex metadata<img width=700/>|An implicit feature ID is assigned to each vertex. The feature table stores `FLOAT64` accuracy values. |![Per-vertex metadata](figures/per-vertex-metadata.png)
+Per-triangle metadata|An implicit feature ID is assigned to each set of three vertices. The feature table stores `FLOAT64` area values.|![Per-triangle metadata](figures/per-triangle-metadata.png)
+Per-point metadata|An implicit feature ID is assigned to each point. The feature table stores `FLOAT64` , `STRING`, and `ENUM` properties, which are not possible through glTF vertex attribute accessors alone.|![Point features](figures/point-features.png)
+Per-node metadata|Vertices in node 0 and node 1 are assigned different `constant` feature IDs. Because the door has articulations these two nodes can't be batched together.|![Per-node metadata](figures/per-node-metadata.png)
+Multi-point features|A point cloud with two feature tables, one storing metadata for groups of points and the other storing metadata for individual points.|![Multi-point features](figures/point-cloud-layers.png)
+Multi-instance features|Instanced tree models where trees are assigned to groups with a per-instance feature ID attribute. One feature table stores per-group metadata and the other stores per-tree metadata.|![Multi-instance features](figures/multi-instance-metadata.png)
+Material classification|A textured mesh using a feature texture to store both material enums and normalized `UINT8` thermal temperatures.|![Material Classification](figures/material-classification.png)
+Composite|A glTF containing a 3D mesh (house), a point cloud (tree), and instanced models (fencing) with three feature tables.|![Composite Example](figures/composite-example.png)
```

Comment:
Will calling this composite confuse readers familiar with composite tiles in 3D Tiles?